src/download/linux.md

```diff
@@ -235,8 +235,12 @@ zypper addrepo https://download.opensuse.org/repositories/home:getchoo/openSUSE_
 zypper refresh
 # stable releases (Qt6 version, only for Tumbleweed)
 zypper install prismlauncher
+# latest builds (Qt6 version, only for Tumbleweed)
+sudo dnf install prismlauncher-nightly
```

Comment:
Opensuse use zypper not dnf from what I remember?

Reply:
copy and paste fail (╯°□°）╯︵ ┻━┻